modules/install-openshift-common-terms.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/index.adoc
+
+:_content-type: REFERENCE
+[id="install-openshift-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the installation content. These terms help you understand installation effectively.
+
+Bootstrap node::
+A temporary machine that runs a minimal Kubernetes configuration to deploy the {product-title} control plane.
+
+Control plane::
+ It is a container orchestration layer that exposes the API and interfaces to define, deploy, and manage the lifecycle of containers. Also known as control plane machines.
+
+Compute node::
+Nodes that are responsible for executing workloads for cluster users. Also known as worker nodes.
+
+Disconnected installation::
+There are situations where parts of a data center might not have access to the internet, even through proxy servers. You can still install the {product-title} in these environments, but you must download the required software and images and make them available to the disconnected environment.
+
+The {product-title} installation program::
+A program that provisions the infrastructure and deploys a cluster.
+
+Installer-provisioned infrastructure::
+The installation program deploys and configures the infrastructure that the cluster runs on.
+
+Ignition config files::
+A file that Ignition uses to configure Red Hat Enterprise Linux CoreOS (RHCOS) during operating system initialization. The installation program generates different Ignition config files to initialize bootstrap, control plane, and worker nodes.
+
+Kubernetes manifests::
+Specifications of a Kubernetes API object in a JSON or YAML format. A configuration file can include deployments, config maps, secrets, daemonsets etc.
+
+Kubelet::
+A **primary node** agent that runs on each node in the cluster to ensure that containers are running in a pod.
+
+Load balancers::
+A load balancer serves as the single point of contact for clients. Load balancers for the API distribute incoming traffic across control plane nodes.
+
+Machine Config Operator::
+An operator that manages and applies configuration and updates of the base operating system and container runtime, including everything between the kernel and kubelet for the nodes in the cluster.
+
+Operators::
+The preferred method of packaging, deploying and managing a Kubernetes application in an {product-title} cluster. It takes human operational knowledge and encodes it into software that is easily packaged and shared with customers.
```

Comment:
deploying, and managing  (Oxford comma)

An Operator takes human operational knowledge